src/main/java/org/sunbird/ratings/model/RatingMessage.java

```diff
@@ -0,0 +1,74 @@
+package org.sunbird.ratings.model;
+
+public class RatingMessage {
```

Comment:
Can we not reuse one bean class instead of RatingMessage and RatingRequest? Both can have the same params right?

Reply:
no they couple of different params for example we are not maintaining version etc.